doc/transform.ipynb

```diff
@@ -2335,8 +2335,14 @@
    "cell_type": "markdown",
```

Comment:
<p>I think the depth-&gt;depth interpolation should actually come first. Its the simpler application in my opinion.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgcm/xgcm/pull/322/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>